packages/firestore/test/unit/specs/describe_spec.ts

```diff
@@ -153,9 +168,10 @@ export function specTest(
     for (const usePersistence of persistenceModes) {
       const spec = builder();
       const runner = getTestRunner(tags, usePersistence);
+      const timeout = getTestTimeout(tags);
       const mode = usePersistence ? '(Persistence)' : '(Memory)';
       const fullName = `${mode} ${name}`;
-      runner(fullName, async () => {
+      const testFunction = runner(fullName, async () => {
```

Comment:
Thoughts on renaming `testFunction` to `queuedTest` or something?  I misunderstood what was going on at first.

Reply:
Renamed